BoxContentSDK/BoxContentSDK/BOXURLRequestSerialization.m

```diff
@@ -1393,7 +1393,7 @@ - (instancetype)initWithCoder:(NSCoder *)decoder {
         return nil;
     }
 
-    self.format = (NSPropertyListFormat)[[decoder decodeObjectOfClass:[NSNumber class] forKey:NSStringFromSelector(@selector(format))] unsignedIntegerValue];
+    self.format = (NSPropertyListFormat)[[decoder decodeObjectOfClass:[NSNumber class] forKey:NSStringFromSelector(@selector((format)))] unsignedIntegerValue];
```

Comment:
Several classes expose methods named `format` with different return types, so just using `selector` here is ambiguous as to which method it's actually referring to. But in this case it seems as though we're just using it to get the property name, so I don't think this is causing any problems. The extra layer of parentheses tells the compiler to ignore the warning.